satpy/readers/ahi_hsd.py

```diff
@@ -255,12 +255,38 @@ class AHIHSDFileHandler(BaseFileHandler):
                             reader_kwargs={'calib_mode':: 'update'})
         scene.load([0.6])
 
+    Alternative AHI calibrations are also available, such as GSICS
+    coefficients. As such, you can supply custom per-channel correction
+    by setting calib_mode='custom' and passing correction factors via:
+       radiance_correction={'chan': ['slo': slope, 'off': offset]}
+    Where slo and off are per-channel slope and offset coefficients defined by:
+     rad_leo = (rad_geo - off) / slo
+    If using custom coefficients, they must be supplied for *all* bands being
+    loaded. If you do not have coefficients for a particular band, you can
+    set slo=1, off=0:
```

Comment:
I think you need `::` here to make the next section code.

Reply:
True, thanks.